src/Scene/Scene.js

```diff
@@ -162,19 +160,24 @@ Scene.prototype.update = function update() {
         }
     });
 
-    // update layers
-    for (const current of config.layerHierarchies) {
-        const layer = current.layer;
+    // loop over layerChains
+    for (const layerLink of config.layerChains) {
+        const layer = layerLink.layer;
+
         // Initial call to get elements to update.
+        // elements is a generic name because its meaning is up to the layer
         const elements = layer.update(context, layer);
 
         if (elements) {
-            updateElements(context, current, elements);
+            updateElements(context, layerLink, elements);
         }
     }
 };
 
 Scene.prototype.step = function step() {
+    this.currentControls().updateCameraTransformation();
```

Comment:
`updateCameraTransformation` is not a good naming.
it updates camera's target  and camera according to the globe